src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ErrorProfileDebugTargetsProvider.cs

```diff
@@ -62,10 +62,11 @@ public Task OnAfterLaunchAsync(DebugLaunchOptions launchOptions, ILaunchProfile
         /// </summary>
         public Task<IReadOnlyList<IDebugLaunchSettings>> QueryDebugTargetsAsync(DebugLaunchOptions launchOptions, ILaunchProfile activeProfile)
         {
-            if (activeProfile.OtherSettings.TryGetValue("ErrorString", out object oErrorString) && oErrorString is string)
+            if (activeProfile.OtherSettings.TryGetValue("ErrorString", out object oErrorString) && oErrorString is string s)
             {
-                throw new Exception(string.Format(VSResources.ErrorInProfilesFile2, Path.GetFileNameWithoutExtension(_configuredProject.UnconfiguredProject.FullPath), (string)oErrorString));
+                throw new Exception(string.Format(VSResources.ErrorInProfilesFile2, Path.GetFileNameWithoutExtension(_configuredProject.UnconfiguredProject.FullPath), s));
             }
```

Comment:
I would prefer "objErrorString" or "errorStringObj" instead of "oErrorString", and "errorString" instead of just "s". In general I dislike single letter variable names as they convey very little information.